### ISSUE TITLE
cuda-devel cannot be installed because cuda-nvml is missing

### ISSUE THREAD

**Issue by zeroepoch** [negativo17/cuda#34]
It appears there is a transition going on from `nvidia-driver-NVML` to `libnvidia-ml`, but this has also caused `cuda-nvml` to go missing. I'm guessing that's because it's not needed anymore? I wonder if the requirements for `cuda-devel` need to be updated?

```
$ sudo dnf install cuda-devel
Last metadata expiration check: 0:32:45 ago on Sun 14 Jul 2024 05:45:41 PM PDT.
Error: 
 Problem: problem with installed package libnvidia-ml-3:555.58.02-2.fc39.x86_64
  - installed package libnvidia-ml-3:555.58.02-2.fc39.x86_64 obsoletes nvidia-driver-NVML < 3:555.58.02-2.fc39 provided by nvidia-driver-NVML-3:555.58-4.fc39.x86_64 from fedora-multimedia
  - package libnvidia-ml-3:555.58.02-2.fc39.x86_64 from fedora-multimedia obsoletes nvidia-driver-NVML < 3:555.58.02-2.fc39 provided by nvidia-driver-NVML-3:555.58-4.fc39.x86_64 from fedora-multimedia
  - package cuda-nvml-devel-1:12.5.82-1.fc39.x86_64 from fedora-multimedia requires cuda-nvml(x86-64), but none of the providers can be installed
  - package cuda-devel-1:12.5.82-1.fc39.x86_64 from fedora-multimedia requires cuda-nvml-devel(x86-64), but none of the providers can be installed
  - conflicting requests
(try to add '--skip-broken' to skip uninstallable packages)
```

**Comment by scaronni** [comment]
Argh. Will fix asap. Thanks for reporting!

**Comment by scaronni** [comment]
Fixed in `555.58.02-3` of the main `nvidia-driver` packge, will be online in an hour or so.

**Comment by zeroepoch** [comment]
Thanks @scaronni! It looks good now and I was able to install `cuda-devel` after upgrading to Fedora 40.
